### ISSUE TITLE
NoClassDefFoundError on Mint/Linux

### ISSUE THREAD

**Issue by paul-griffith** [paul-griffith/kindling#33]
```
WARNING: package com.sun.java.swing.plaf.windows not in java.desktop                 
Exception in thread "AWT-EventQueue-0" java.lang.NoClassDefFoundError: com/sun/java/swing/plaf/windows/WindowsLookAndFeel
```
https://forum.inductiveautomation.com/t/kindling-a-collection-of-tools-to-help-with-ignition/54689/12?u=pgriffith

Needs investigation; JDK should be JDK but FlatLaf may have implicit dependencies.

**Comment by paul-griffith** [comment]
Or maybe it's Jide

**Comment by paul-griffith** [comment]
This is Jide; turns out it's fixed but Jide is no longer building artifacts themselves, so need to switch dependencies to Formdev's fork:
https://github.com/jidesoft/jide-oss/issues/48
https://github.com/jidesoft/jide-oss/issues/37

**Comment by paul-griffith** [comment]
Fixed in bcac970306e9eec3708a492fbea895d8acd5fac7
